### ISSUE TITLE
Update button styles

### ISSUE THREAD

**Issue by yaili** [ubuntudesign/vanilla-framework#354]
- Buttons to follow Juju buttons design, except the Juju deploy button
- The brand button (eg. orange button in Ubuntu) is called **cta**

**Comment by jamiedawsonyoung** [comment]
Quick question… Do we have a minimum width for buttons? I've got a "Add" button in WebDM now and it's going to look quite narrow. I've tried to implement a min size for buttons in client but not sure we've covered it here? (maybe there's a rule in place already?)

**Comment by yaili** [comment]
I think that's a question for @spencerbygraves I'm not sure what the rules are in Juju but that's the pattern we're following.

**Comment by spencerbygraves** [comment]
I've been using our rule from vanilla, where padding is based on the font size. 0.75 above and below, 1.5 left and right. I haven't needed to go smaller than that so far.

**Comment by jamiedawsonyoung** [comment]
So, applying these rules (which I've been doing too) gives you this: 

![screenshot 2016-05-18 13 05 58](https://cloud.githubusercontent.com/assets/6909617/15358022/20a64ff4-1cfa-11e6-9cfa-3cb5a694707d.png)

Which is okay… but I know there's even shorter labels in Maas like "OK" so maybe we should come up with a min-size?

**Comment by spencerbygraves** [comment]
Sure, sounds like a minimum size would be useful

**Comment by jamiedawsonyoung** [comment]
@yaili What do you think? Are there any rules in place at the moment that govern buttons on the web?

**Comment by yaili** [comment]
Sounds like something maybe we should look into.

**Comment by jamiedawsonyoung** [comment]
👍

**Comment by barrymcgee** [comment]
Related: #191

**Comment by barrymcgee** [comment]
@anthonydillon Is this related to the buttons work you were talking about?

**Comment by jamiedawsonyoung** [comment]
Just a wee side note… 

Had an interesting chat with some of the platform team about a max width button rule we'd suggested. It was mainly to avoid having too many of the button sausages that pepper the phone interface (the full width ones) 

We decided in the end not to cap the width as it does more for bad user experience than it does for teaching developers to write more succinct button labels. Our approach will be more in a guidance than built in button rules.

We also discussed a "almost the same so make them the same" rule for buttons where if two buttons are a similar size (with a set tolerance) then the system would make them the same. So if you had a 100px button and a 105px button then the system would make them both 105px and centre the label in both. I've no idea if that's even possible (or sensible) for the web? It's also a bit of a long way off for us too, we still need to get the standard buttons landed…
